ivy/functional/frontends/torch/tensor.py

```diff
@@ -1048,6 +1048,9 @@ def count_nonzero(self, dim):
     def exp(self):
         return torch_frontend.exp(self)
 
+    def expm1(self):
```

Comment:
Hi! Thanks for contributing, but tests are failing on the CI and this is a change that needs to be made similar to other torch functions as bfloat16 and float16 are not supported for torch and this decorator needs to be added to mark it as unsupported. 

Feel free to request for another review once you've made this change.
```suggestion
@with_unsupported_dtypes({"2.0.1 and below": ("bfloat16", "float16")}, "torch")
    def expm1(self):
```

Reply:
Added the unsupported types